scripts/src/main/resources/scripts/functions

```diff
@@ -1381,7 +1397,7 @@ function doInstall() {
     install_path="${DEVON_SOFTWARE_PATH}/${repo}/${software}/${version}"
     version_file="${install_path}/.devon.software.version"
   fi
-  if [ ! -d "${install_path}" ] || [ "${install_path}" == "${target_path}" ]
+  if [ ! -d "${install_path}" ] || [ "${install_path}" == "${target_path}" ] && [ "$isOnline" = 0 ]
```

Comment:
if a tool is not installed at all and I will run devon command to install it, with this change, the command will succeed but do nothing. This is not the expected behavior for the end-user.